README.md

```diff
@@ -1,20 +1,67 @@
+> [!IMPORTANT]
+> ThreadSharp is limited by the capabilities of the [Threads public API](https://developers.facebook.com/docs/threads/) and so not all features that you enjoy in the Threads app are available.
+
 ![ThreadSharp banner](https://github.com/itsWindows11/ThreadSharp/blob/main/assets/banner.png?raw=true)
+<h4 align="center"> A C# API wrapper for the Threads API.</h4>
+<p align="center">
+  <a style="text-decoration:none" href="https://github.com/itsWindows11/ThreadSharp/actions/workflows/nuget.yml">
+    <img src="https://github.com/itsWindows11/ThreadSharp/actions/workflows/nuget.yml/badge.svg" alt="CI Status" /></a>
+  <a style="text-decoration:none" href="https://discord.com/channels/714581497222398064/913496895006072843">
+    <img src="https://img.shields.io/badge/discuss-on%20discord-5865F2" alt="Discord" /></a>
+  <a style="text-decoration:none" href="https://www.nuget.org/packages/ThreadSharp">
+    <img src="https://img.shields.io/nuget/v/ThreadSharp" alt="Discord" /></a>
+</p>
 
-# <center>ThreadSharp</center>
-<center>A C# API wrapper for the Threads API.</center>
+## Documentation
 
-![NuGet Version](https://img.shields.io/nuget/v/ThreadSharp)
+You can browse the documentation at [itswindows11.github.io/ThreadSharp](https://itswindows11.github.io/ThreadSharp/docs/).
 
-## Documentation
+## Contributing
 
-You can browse the documentation at https://itswindows11.github.io/ThreadSharp.
+There are multiple ways to participate in the community:
+
+- Upvote popular feature requests
+- [Submit a new feature](https://github.com/itsWindows11/ThreadSharp/pulls)
+- [File bugs and feature requests](https://github.com/itsWindows11/ThreadSharp/issues/new/choose).
+- Review source [code changes](https://github.com/itsWindows11/ThreadSharp/commits)
 
 ## Usage
 
+> [!NOTE]
+> ThreadSharp doesn't handle authentication manually, so you will have to handle the authentication depending on your platform's OAuth2 library.
+
 You can start by initializing a `ThreadsClient` with an access token:
 
 ```c#
 ThreadsClient threadsClient = new ThreadsClient("access-token");
 ```
 
-Note that ThreadSharp doesn't handle authentication manually, so you will have to handle the authentication depending on your platform's OAuth2 library.
+## Building the Code
+
+### Prerequisites
+
+Ensure you have following components:
+
+- [Git](https://git-scm.com/)
+- a .NET IDE of your choice, preferably [Visual Studio 2022](https://visualstudio.microsoft.com/vs/)
+- the .NET SDK
+
+### Git
+
+Clone the repository:
+
+```git
+git clone https://github.com/itsWindows11/ThreadSharp
+```
+
+### Build the project
+
+- Open `ThreadSharp.sln`.
+- Set the Startup Project to `ThreadSharp.csproj` (of course!)
```

Comment:
Library projects cannot be used as startup projects IIRC.

Reply:
Oops, I just copied that section from a template 🥴